README.md

```diff
@@ -365,3 +364,14 @@ to retrieve the OCSP status of a certificate, run Pebble with a `pebble-config.j
 ```
   "ocspResponderURL": "http://127.0.0.1:4002",
 ```
+
+### Listing orders
+
+[As opposed to Boulder](https://github.com/letsencrypt/boulder/issues/3335), Pebble has support
```

Comment:
I think its probably not necessary to discuss Boulder here. It will be one more place to remember to update if/when we resolve Boulder issue 3335.

Reply:
I mainly wanted to make sure that people remember that just because Pebble can do this, it doesn't mean that Boulder will also return `orders` :) But I can remove it.